maitake-sync/Cargo.toml

```diff
@@ -17,7 +17,7 @@ categories = [
     "concurrency"
 ]
 edition = "2021"
-rust-version = "1.61.0"
+rust-version = "1.76.0"
```

Comment:
I don't actually think we need to do the MSRV bump for `maitake-sync`. Since the `cordyceps` changes will be released in a semver-compatible version, and `maitake-sync`'s minimum `cordyceps` version isn't changing, users of `maitake-sync` can continue using the current cordyceps version rather than picking up the one with the new MSRV --- either by checking in a lockfile, or by explicitly pinning the `cordyceps` version in their Cargo.toml. 

IMHO, we would only need to change `maitake-sync`'s MSRV if we were changing the dependency spec for `cordyceps` to not permit versions that compile on 1.61.0.